src/components/patientPage/patientCard/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+
+import { Callout, Icon, H3, H5 } from "@blueprintjs/core";
+import PatientInfo from "components/patientPage/patientCard/patientInfo";
+import { Patient } from "types";
+import { getPatientResources, getSubjectResources } from "services/api";
+
+import "./style.css";
+
+interface Props {
+  patient: Patient;
+}
+
+const PatientCard = ({ patient }: Props) => {
+  /*
+  getPatientNumberCard and getSubjectNumberCard are now rendering PatientInfo elements with click option which print the results on the console.
+  */
+  const getPatientNumberCard = (resourceName: string, writtenName: string) => {
+    if (patient.number !== undefined)
+      if (patient.number[resourceName] !== undefined)
+        return (
+          <div
+            onClick={async () => {
+              const response = await getPatientResources(
+                resourceName,
+                patient.id
+              );
+              console.log(writtenName + " : ", response.data.entry);
```

Comment:
why don't you save the response anywhere and just print it to the console ?

Reply:
Now saved in patient attributes